File: Quick_Sort/Haskell/NorfairKing/Quicksort_test.hs
module Quicksort_test (main) where
import Quicksort ( quicksort )

main = do
    let list = [1..5] ++ [1,3,4,6,2]  ++ [5..1]
    putStrLn "Unsorted: "
    putStrLn (show list)
    putStrLn ""
    putStrLn "|"
    putStrLn "v"
    putStrLn ""
    putStrLn "Sorted: "
    putStrLn $ (show . quicksort) list 

File: Quick_Sort/Scala/timleathart/QuickSort_test.scala
object QuickSort_Test {

  def main(args: Array[String]): Unit = {
    val list = List(5,2,5,7,8,1,2,4,6,8,2,3)    
    println("Before: " + list.mkString(", "))
    
    val sorted = QuickSort.QuickSort(list);
    println("After: " + sorted.mkString(", "))
  }

}
